clients/task/task.go

```diff
@@ -52,9 +54,217 @@ func (c Client) Create(ctx context.Context, params *CreateParams) error {
 	})
 }
 
+type FindParams struct {
+	clients.OrgParams
+	TaskID string
+	UserID string
+	Limit  int
+}
+
+func (c Client) Find(ctx context.Context, params *FindParams) error {
+	org, err := c.getOrg(&params.OrgParams)
+	if err != nil {
+		return err
+	}
+	if params.Limit < 1 {
+		return fmt.Errorf("must specify a positive limit, not %d", params.Limit)
+	}
+
+	var tasks []api.Task
+	// If we get an id, just find the one task
+	if params.TaskID != "" {
+		task, err := c.GetTasksID(ctx, params.TaskID).Execute()
+		if err != nil {
+			return err
+		}
+		tasks = append(tasks, task)
+	} else {
+		// filter on all tasks
+		taskGet := c.GetTasks(ctx).Org(org).Limit(int32(params.Limit))
```

Comment:
Is the task API one of the "smart" ones that can handle either an ID or a name in the `org` parameter? The swagger spec doesn't mention it (but I wouldn't be shocked to learn that it's been implemented)

Reply:
No I don't believe it is - is that important?

Reply:
Yes, I think it'll fail as-is if somebody passes `--org-id`. I think there needs to be a conditional that checks:
* If `--org-id` given, set `OrgId`
* Else set `Org` using either `--org` or the config profile

Reply:
You're right - testing showed this was broken. It should be good now - see the new commit.